frontend/src/app/commonComponents/DateInput.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
I've been replacing these with the `uniqueIds` that's in my PR. Generating a true UUID is pretty expensive and all we need here is something unique to a particular rendering of the component. If these components are used in snapshot tests we need them to have reproducible id values or they'll generate diffs. If it's easier we can just merge this and update it once my PR lands.

Reply:
This `DateInput` isn't used atm; it doesn't have on-par validation and error handling compared to the CMSDS one. 

History:
I was originally using https://www.npmjs.com/package/react-html-id, which mostly worked and generated reproducable ids for snapshots. It solved the problem for dynamically generated lists of components, but did not work for GUIDs. For example, `patients-${id}` would conflict with other components rendered at the same time that also coincidentally used `patients-${id}`, since `id` was not a global value and would increment up from 0 in each component. I didn't want the mental overhead of ensuring that prefixes (ie: `patients-`) were globally unique throughout the app, so i switched to `uuid` as a quick fix. 

There is probably some config for this to ensure global unique values, but the library was also designed for class components, not functional components.

Conclusion:
I support stripping this out in favor of your implementation